meshio/medit/_medit.py

```diff
@@ -47,7 +48,10 @@ def read_buffer(f):
             break
 
         line = line.strip()
-        if len(line) == 0 or line[0] == "#":
+        if len(line) == 0:
+            continue
+        if line[0] == "#":
+            spec_info.append(line[1:])
```

Comment:
Let's just call this `info`, not `spec_info`.

Reply:
I started with XDMF `time_series.py` and there was already one `info` variable, so I used `spec_info`. And to be consistent, it is also in XDMF/main.py, MEDIT and VTK.
- `spec_info` in XDMF `time_series.py` and `info` everywhere else?
- `spec_info` everywhere?